ledger-service/http-json/src/it/scala/http/WebsocketServiceIntegrationTest.scala

```diff
@@ -450,6 +450,80 @@ class WebsocketServiceIntegrationTest
       } yield resumes.foldLeft(1 shouldBe 1)((_, a) => a)
   }
 
+  "fetch multiple keys should work" in withHttpService { (uri, encoder, _) =>
+    def matches(expected: Seq[JsValue], actual: Seq[JsValue]): Boolean =
```

Comment:
As far as I can tell the only difference between `==` and this is that you ignore extra keys for objects. Is that intentional? If so why? If not, why not use `==`?